NewForce-at-Mountwest/bangazon-api-bucatini
# Russ controller payment

# Description

Controller and Test files for Payment Type.


## Related Ticket(s)
Ticket 3


## Steps to Test
Use browser (ctrl-f5) to check 
     -Get all Payment Types where Archived = false
     -Get Single Payment Types

Use Postman to check
     -Get all
     -get single
     -Post(without 'Id' or 'Archived' properties)
     -Put(without 'Id' or 'Archived' properties)
     -Soft Delete(example: "https://localhost:5001/api/PaymentType/2") should only change archived 
            status to true
     -Hard Delete (example: "https://localhost:5001api/PaymentType/3?Delete=true") Should Hard Delete



# Checklist:
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new errors
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
